CONFIGURATION_OPTIONS.md

```diff
@@ -223,9 +223,9 @@ Example:
 ```
 
 ##### `globalVarName` [Optional]
+**Usage of this parameter is discouraged and it will be deprecated in a future version of this module.** This parameter allows to define the name for the LiveConnect instance that is attached to the window object. If none is provided, Liveconnect will not be attached to the window object. 
```

Comment:
alternatively, let's keep using liQ as the default for now but allow passing null explicitly to disable attaching.

(though of course I'm in favour of nuking it asap :D)

Reply:
Let's nuke it.